### ISSUE TITLE
Latest version does not show up when using PIP

### ISSUE THREAD

**Issue by jmahmood** [heroku/kafka-helper#5]
I get the following error when trying to load the version of the library from pip.

```
>>> import kafka_helper
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jmahmood/LunchMePython36/lib/python3.6/site-packages/kafka_helper.py", line 9, in <module>
    from urlparse import urlparse
ModuleNotFoundError: No module named 'urlparse'
>>> from confluent_kafka import Producer
```

I think this issue has already been fixed in the repo, but the fixed version is not available on pip.  I think you need to push out a new version.  

Also, is cryptography==1.5 a strict necessity?  Installing the latest version is not a problem, but older versions seem to have issues on the mac.

**Comment by jmahmood** [comment]
@lambacck FYI

**Comment by apotapov** [comment]
Installing with `pip install -e git+https://github.com/heroku/kafka-helper@70aa539540e7d6517f0c3c1900617767ab198b6c#egg=kafka_helper` does the trick.

**Comment by alouie-sfdc** [comment]
Thanks -- this should be fixed with the release of kafka-helper 0.2: https://pypi.org/project/kafka-helper/0.2/
